lib/service/skills/OccupationsService.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:illinois/model/occupation/Occupation.dart';
+import 'package:illinois/service/skills/IOccupationsData.dart';
+// import 'package:illinois/service/skills/MockOccupationsData.dart';
+import 'package:illinois/service/skills/OnetOccupationsData.dart';
+
+class OccupationsService /* with Service */ {
```

Comment:
In order to add `OccupationsService` to the list in `mainImpl.dart` (as I suggested in another comment), you will need to uncomment `with Service`.